RealestateApi/Controllers/RealEstateController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using RealEstateApi.Dto.Request;
+using RealEstateApi.Dto.Response;
+using RealEstateApi.Service;
+using RealEstateApi.Service.Interfaces;
+
+namespace RealEstateApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class RealEstateController : ControllerBase
+    {
+        private readonly ILogger<RealEstateController> _logger;
+        private readonly IRealEstateService _realEstateService;
+
+        public RealEstateController(ILogger<RealEstateController> logger, IRealEstateService realEstateService)
+        {
+            _logger = logger;
+            _realEstateService = realEstateService;
+        }        
+
+        [HttpGet(Name = "GetRealEstate")]
+        public async Task<RealEstateResponseDto> GetRealEstate() 
```

Comment:
The name of the method should be: GetAllRealEstates because in the future we'll have to have a Get Real State By Id